site/content/docs/4.3/migration.md

```diff
@@ -35,6 +35,9 @@ Changes to our source Sass files and compiled CSS.
 - Removed print styles and `$enable-print-styles` variable. Print display classes, however, have remained intact. [See #28339](https://github.com/twbs/bootstrap/pull/28339).
 - Dropped `color()`, `theme-color()` & `gray()` functions in favor of variables. [See #29083](https://github.com/twbs/bootstrap/pull/29083)
 - The `theme-color-level()` function is renamed to `color-level()` and now accepts any color you want instead of only `$theme-color` colors. [See #29083](https://github.com/twbs/bootstrap/pull/29083)
+- Breakpoint media queries changes [See #29148](https://github.com/twbs/bootstrap/pull/29148)
```

Comment:
I'd add an example here to make it easier for people to upgrade

Reply:
Do the bullet points below not explain this enough, or any suggestions how to add examples here?

Reply:
I'd expand the bullet points with examples in each case

Reply:
Or just an example per case

Reply:
Done